### DISCUSSION TITLE
Baggage MDC propagation in Webflux

### DISCUSSION THREAD

**Discussion by bikerp** [open-telemetry/opentelemetry-java-instrumentation#10112] [Q&A]
I'm using otel.instrumentation.logback-mdc.add-baggage settings to add baggage into logs. I can see the baggage in logs for the same thread. However the baggage is not propagated to Webclient threads. Sounds like a bug to me.

**Comment by trask** [comment]
hi @bikerp, can you open an issue and provide some more details? is normal context propagated (e.g. `traceparent` header)? can you provide a repro that we can use to investigate further?

**Reply by bikerp** [reply]
Hi @trask , I have created issue #10127 and provided sample code
